mod/notifications.php

```diff
@@ -79,13 +78,14 @@ function notifications_content(App $a)
 	}
 
 	$page = ($_REQUEST['page'] ?? 0) ?: 1;
-	$show =  $_REQUEST['show'] ?? 0;
+	$show =  ($_REQUEST['show'] ?? '' === 'all');
```

Comment:
This is equivalent to `isset($_REQUEST['show']) ? $_REQUEST['show'] : false`.

Parentheses should enclose the null coalesce operator:
```suggestion
	$show =  ($_REQUEST['show'] ?? '') === 'all';
```